andrew/euler/euler_3_1.py

```diff
@@ -0,0 +1,18 @@
+'''Find the largest prime factor of the number 600851475143'''
+
+# Enter number from input
+a = int(input("Enter number to find its largest prime factor: "))
+
+# Check top down to make it faster
+for x in range(a, 1, -1):
+    if a % x == 0:
+        factor = x
+        print(factor)
+        for i in range(2, factor):
```

Comment:
Avoid nested list inside an if block. It's complex. Signal intent using function names and variable names.